README.md

```diff
@@ -51,6 +51,27 @@ If you want to share Whatsapp and Mailto, you should write `LSApplicationQueries
     ```
 8. Run your project (`Cmd+R`)
 
+
+#### iOS Install(using Pods)
+
+If you wish, you can use [cocopoads](https://cocoapods.org/) to use react-native-share.
+
+You just need to add to your Podfile thr react-native-share dependencie.
+
+```ruby
+  # React-Native-Share pod
+  pod 'RNShare', :path => '../node_modules/react-native-share'
+```
+
+After that, just run a `pod install` or `pod udpate` to get up and running with react-native-share. 
+
+Then run a `react-native link react-native-share`, and doing the steps 6 and 7.
+
+You can also see our example to sse how you need to setup your podfile.
+
+Btw, i also recommend reading this (amazing article)[https://shift.infinite.red/beginner-s-guide-to-using-cocoapods-with-react-native-46cb4d372995] about how pods and rn. =D
```

Comment:
typo on `i`?
maybe it's better `I` or `We`

Reply:
fixed